pennylane_lightning/lightning_kokkos/lightning_kokkos.toml

```diff
@@ -1,124 +1,105 @@
+MultiRZ                = { properties = [ "invertible",                 "differentiable" ] }
+PauliX                 = { properties = [ "invertible",                 "differentiable" ] }
+PauliY                 = { properties = [ "invertible",                 "differentiable" ] }
+PauliZ                 = { properties = [ "invertible",                 "differentiable" ] }
+PhaseShift             = { properties = [ "invertible",                 "differentiable" ] }
+QubitUnitary           = { properties = [ "invertible",                                  ] }
+Rot                    = { properties = [ "invertible",                                  ] }
+RX                     = { properties = [ "invertible",                 "differentiable" ] }
+RY                     = { properties = [ "invertible",                 "differentiable" ] }
+RZ                     = { properties = [ "invertible",                 "differentiable" ] }
+SingleExcitationMinus  = { properties = [ "invertible",                 "differentiable" ] }
+SingleExcitationPlus   = { properties = [ "invertible",                 "differentiable" ] }
+SingleExcitation       = { properties = [ "invertible",                 "differentiable" ] }
+S                      = { properties = [ "invertible",                 "differentiable" ] }
+SWAP                   = { properties = [ "invertible",                 "differentiable" ] }
+Toffoli                = { properties = [ "invertible",                 "differentiable" ] }
+T                      = { properties = [ "invertible",                 "differentiable" ] }
 
 # Operators that should be decomposed according to the algorithm used
 # by PennyLane's device API.
 # Optional, since gates not listed in this list will typically be decomposed by
 # default, but can be useful to express a deviation from this device's regular
 # strategy in PennyLane.
-decomp = [
-        "BasisState",
-        "QubitStateVector",
-        "StatePrep",
-        "QFT",
-        "MultiControlledX",
-]
+[operators.gates.decomp]
+
+BasisState             = {}
+MultiControlledX       = {}
+QFT                    = {}
+QubitStateVector       = {}
+StatePrep              = {}
 
 # Gates which should be translated to QubitUnitary
-matrix = [
-        "ControlledQubitUnitary",
-        "ECR",
-        "SX",
-        "ISWAP",
-        "PSWAP",
-        "SISWAP",
-        "SQISW",
-        "CPhase",
-        "OrbitalRotation",
-        "QubitCarry",
-        "QubitSum",
-        "DiagonalQubitUnitary",
-        "BlockEncode",
-]
+[operators.gates.matrix]
+
+BlockEncode            = {}
+CPhase                 = {}
+DiagonalQubitUnitary   = {}
+ECR                    = {}
+ISWAP                  = {}
+OrbitalRotation        = {}
+PSWAP                  = {}
+QubitCarry             = {}
+QubitSum               = {}
+SISWAP                 = {}
+SQISW                  = {}
+SX                     = {}
+
+# Observables supported by the device
+[operators.observables]
```

Comment:
The case of `"Sum", "SProd", "Prod", "Exp"` is interesting. I see why they were removed (there is no native API for them), but Catalyst will convert them to the old style observables after the tape is processed, so we'll have to remember in our decomposition/verification functions that they are not flagged unsupported unless the device also doesn't support the equivalent old style (Hamiltonian, Tensor, etc) observable.